<launch>
	<arg name="quad" default="SQ01s"/>
	<arg name="gui" default="true"/>
	<arg name="rviz" default="true"/>
	<arg name="environment" default="true"/>
	<!-- Start faster commands 
	<node ns="$(arg quad)" name="faster_commands" pkg="faster" type="faster_commands.py" output="screen"/>
	 -->
	<!-- Start Behaviour Selector-->
	  <node pkg="behavior_selector" type="behavior_selector_node.py" name="behavior_selector" output="screen" />


  <!-- Don't run this, for instance, for ssh connections -->
<group if="$(arg rviz)">

	<!-- Start Rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find faster)/rviz_cfgs/cvx_all.rviz"/>

</group>

  <!-- Don't run this, for instance, for ssh connections -->
<group if="$(arg gui)">
	<!-- Start GUI 	-->
	<node pkg="rqt_gui" type="rqt_gui" name="rqt_gui" args="--perspective-file $(find faster)/rqt_cfgs/demo_interface.perspective"/>

</group>

<group if="$(arg environment)">
    <!-- Start dynamic obstacles -->
	<node name="dynamic_obstacles" pkg="faster" type="dynamic_forest.py" output="screen"/>
</group>s	

</launch>
